tob-web/openshift/angular-on-nginx-deploy.test.param

```diff
@@ -6,12 +6,12 @@
 # NAME=angular-on-nginx
 # IMAGE_NAMESPACE=devex-von-tools
 # REAL_IP_FROM=172.51.0.0/16
-# API_URL=https://django-devex-von-dev.pathfinder.gov.bc.ca/api/V2/
+# API_URL=https://dev-demo-api.orgbook.gov.bc.ca/api/V2/
```

Comment:
Should this be `demo-api.orgbook.gov.bc.ca/api/V2/`, since it is for the `test` environment?

Reply:
yes

Reply:
updated